hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -16,9 +19,25 @@
   interrupt_list: [
     { name: "es_entropy_valid"
       desc: "Asserted when entropy source bits are available."}
-    { name: "es_entropy_fifo_err"
+    { name: "es_ana_src_ok"
+      desc: "Asserted when the analog source is available."}
+    { name: "es_ana_bits_err"
+      desc: "Asserted when the analog source data bits have a polarity error."}
```

Comment:
when you say polarity, do you mean the differential error..?

Reply:
Yes, will change if that is clearer.

Reply:
I think that might help, because I believe it is referred to differential on the other side.